src/utils/ammOperations.ts

```diff
@@ -0,0 +1,21 @@
+import { OpStatus } from 'ergo-dex-sdk/build/main/amm/models/ammOperation';
+
+import { RefundParams } from 'ergo-dex-sdk/build/module/models/refundParams';
+import { TransactionContext } from 'ergo-dex-sdk/build/module/ergo/wallet/entities/transactionContext';
+import { ammOrderRefunds } from './ammOrderRefund';
+
+const validRefundStatuses = ['pending', 'submitted'];
+
+export const isRefundableOperation = (status: OpStatus): boolean =>
+  validRefundStatuses.indexOf(status) !== -1;
+
+export const refund = async (
```

Comment:
I think this function should contain all refund-related operations(box selection, etc) to separate UI from refund logic. 
I suggest moving those operations here to keep in the refund modal only the result handling.